puppet/zulip/files/postgresql/process_fts_updates

```diff
@@ -123,6 +125,12 @@ if "host" in pg_args:
     else:
         pg_args["sslmode"] = "verify-full"
     pg_args["connect_timeout"] = "600"
+elif get_config(get_config_file(), "postgresql", "database_name") != "zulip":
+    pg_args["dbname"] = settings.DATABASES["default"]["NAME"]
+    if settings.DATABASES["default"]["USER"] != "zulip":
+        pg_args["user"] = settings.DATABASES["default"]["USER"]
+        pg_args["password"] = settings.DATABASES["default"]["PASSWORD"]
+        pg_args["host"] = settings.DATABASES["default"]["HOST"]
```

Comment:
I don't understand why these are nested under the check for the database name.  That implies that we don't support changing the _user_ if we're not changing the _database name_, which is an odd limitation.  Can we uncouple those?

It's worth noting that this, like lines 95-99, has the problem that it may be run on a database server without access to the Django `settings` module.  In such a world, `settings` will be unset, since the import on line 93 failed.  The block above, on lines 117-127 is "safe" from this because the only way to get a `"host" in pg_args` is if it was set on line 96, which means the import succeeded.